tests/Client/UpdaterTest.php

```diff
@@ -254,6 +273,7 @@ public function testFileNotFoundExceptions(string $fixturesSet, string $fileName
         // side-effects.
         $this->localRepo = $this->memoryStorageFromFixture($fixturesSet, 'tufclient/tufrepo/metadata/current');
         $this->testRepo = new TestRepo($fixturesSet);
+        $this->assertClientRepoVersions($this->getFixtureClientStartVersions($fixturesSet));
```

Comment:
Shouldn't this be using static::getFixtureClientStartVersions()?

Reply:
yep fixed